### PR TITLE
use namespace labels to prevent SCC usage

### PR DESCRIPTION
**Pull request by deads2k**
Use namespace labels to prevent SCC usage.  This will prevent using openshift APIs when they may not be available.

/assign @derekwaynecarr 
/cc @abhinavdahiya

### CODE REVIEW

File: test/extended/util/test.go [hunk 1]
```diff
@@ -206,6 +206,12 @@ func createTestingNS(baseName string, c kclientset.Interface, labels map[string]
 		return fn(baseName, c, labels)
 	}
 
+	// if we're running an upstream test, make sure to skip openshift admission.
+	// TODO we may be able to relax this after the openshift apiserver is back in the openshift/installer
```
Comment (derekwaynecarr) [inline test/extended/util/test.go]: bz to track that we dont forget to come back to this?

Comment (deads2k) [reply test/extended/util/test.go]: > bz to track that we dont forget to come back to this?

I'm actually not sure. do we want to?

**Comment by deads2k** [comment]
> e2es have never passed on this

fixed

**Pull request merged**